homeassistant/components/stream/worker.py

```diff
@@ -45,9 +45,9 @@ def __init__(
         self._memory_file: BytesIO = cast(BytesIO, None)
         self._av_output: av.container.OutputContainer = None
         self._input_video_stream: av.video.VideoStream = None
-        self._input_audio_stream = None  # av.audio.AudioStream | None
+        self._input_audio_stream: Any | None = None  # av.audio.AudioStream | None
```

Comment:
It actually works for me, please try it out. Otherwise you can use `av.audio.stream.AudioStream`.
```suggestion
        self._input_audio_stream: av.audio.AudioStream | None = None
```

Reply:
Neither of those will pass pylint checks

Reply:
@cdce8p already noted this issue when adding #49103 .